build.cake

```diff
@@ -22,156 +20,78 @@ var signingCertificatePassword = Argument("signing_certificate_password", "");
-
-    Information("Finished running tasks for build v{0}", nugetVersion);
+    Information("Finished running tasks.");
 });
 
 //////////////////////////////////////////////////////////////////////
 //  PRIVATE TASKS
 //////////////////////////////////////////////////////////////////////
 
-Task("__Clean")
+Task("Clean")
     .Does(() =>
 {
     CleanDirectory(artifactsDir);
     CleanDirectories("./**/bin");
     CleanDirectories("./**/obj");
 });
 
-Task("__Restore")
-    .Does(() => DotNetCoreRestore());
-
-Task("__UpdateAssemblyVersionInformation")
-    .Does(() =>
-{
-    foreach (var project in projectsToPackage)
-    {
-        var assemblyInfoFile = Path.Combine(sourceFolder, project, "Properties", "AssemblyInfo.cs");
-        RestoreFileOnCleanup(assemblyInfoFile);
-        GitVersion(new GitVersionSettings {
-            UpdateAssemblyInfo = true,
-            UpdateAssemblyInfoFilePath = assemblyInfoFile
-        });
-    }
-    
-    Information("AssemblyVersion -> {0}", gitVersionInfo.AssemblySemVer);
-    Information("AssemblyFileVersion -> {0}", $"{gitVersionInfo.MajorMinorPatch}.0");
-    Information("AssemblyInformationalVersion -> {0}", gitVersionInfo.InformationalVersion);
-});
-
-Task("__Build")
-    .Does(() =>
-{
-    var settings =  new DotNetCoreBuildSettings
-    {
-        Configuration = configuration
-    };
-
-    if(!string.IsNullOrEmpty(framework))
-        settings.Framework = framework;      
-
-    DotNetCoreBuild("**/project.json", settings);
-});
+Task("Restore")
+	.IsDependentOn("Clean")
+    .Does(() => DotNetCoreRestore("source"));
 
-Task("__ZipNET45TestProject")
+Task("Build")
+    .IsDependentOn("Restore")
     .Does(() => {
-        Zip("source/Calamari.Tests/bin/Release/net451/win7-x64/", Path.Combine(artifactsDir, "Binaries.zip"));
-    });
-
-Task("__Test")
-    .Does(() =>
-{
-    var settings =  new DotNetCoreTestSettings
-    {
-        Configuration = configuration
-    };
-
-    if(!string.IsNullOrEmpty(framework))
-        settings.Framework = framework;  
-
-    if(!string.IsNullOrEmpty(testFilter))
-        settings.ArgumentCustomization = f => {
-            f.Append("-where");
-            f.AppendQuoted(testFilter);
-            return f;
-        };
-
-     DotNetCoreTest("source/Calamari.Tests/project.json", settings);
-});
-
-Task("__Pack")
-    .Does(() =>
-{
-    DoPackage("Calamari", "net40", nugetVersion);
-    DoPackage("Calamari.Azure", "net45", nugetVersion);   
-});
-
-private void DoPackage(string project, string framework, string version)
-{ 
-	var outputDirectory = Path.Combine(artifactsDir, project);
-	
-    DotNetCorePublish(Path.Combine("./source", project), new DotNetCorePublishSettings
-    {
-        Configuration = configuration,
-        OutputDirectory = outputDirectory,
-        Framework = framework,
-		NoBuild = true
-    });
-
-	SignBinaries(outputDirectory);
+		 DotNetCoreBuild("./source", new DotNetCoreBuildSettings
+		{
+			Configuration = configuration

```

Comment:
Ah we are using Attribute generation, in that case we don't then :). If you set the version here though, dotnet will generate the attributes for you.